src/Gaufrette/Adapter/AwsS3.php

```diff
@@ -111,10 +111,12 @@ public function rename($sourceKey, $targetKey)
 
         try {
             $this->service->copyObject($options);
-            return true;
         } catch (\Exception $e) {
             return false;
         }
+        $this->delete($sourceKey);
```

Comment:
@staabm I guess that refers to the content of the file, not the atomicity of the copy-then-delete operation, right?